resource_booking/__manifest__.py

```diff
@@ -25,7 +25,6 @@
         "mail",
         "portal",
         "resource",
-        "web_calendar_slot_duration",
```

Comment:
You can't remove this dependency, it is necessary to maintain the functionallity of the module. Here I post a PR that i made migrating correctly the module: https://github.com/OCA/web/pull/2646